frontend/components/Fields/HttpsUrlField.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { isURL } from 'validator';
+
+import RenderUrlField from './RenderUrlField';
+
+
+export const isHttpsUrlWithoutPath = (value) => {
+  if (value && value.length) {
+    if (!isURL(value, { protocols: ['https'] }) || value.lastIndexOf('/') !== 7) {
```

Comment:
Does this method report a valid URL if the two trailing `/` after the `:` are missing?

Reply:
Nope, it reports that is invalid. Just added a couple additional test expectations to show that in 69/commits/4811c

Reply:
if that's the case, do we need the `||` condition here?

Reply:
that's to make sure that there is no additional path in the url. `7` is the position of the last `/` in `https://`. Any additional would indicate that there is a sub-path, which we don't want.

ie, `https://example.gov` is valid, but `https://example.gov/` and `https://example.gov/whatever` are not valid.

Reply:
I didnt see the withoutPath part of the method name, makes sense.